src/index.html

```diff
@@ -1,16 +1,54 @@
-<!doctype html>
-<html lang="en">
-  <head>
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head class="page__head">
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.scss">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+
+  <body class="page__body">
+    <header class="header"></header>
+
+    <main class="main">
+      <form
+        class="search__form search__form--big"
```

Comment:
please give correct naming for BEM blocks, as it is a block, not block element. fix it everywhere
```suggestion
        class="search-form search-form--big"
```